src/go/configinfo/service_info.go

```diff
@@ -361,9 +370,13 @@ func (s *ServiceInfo) processHttpRule() error {
 			method := s.Methods[r.GetSelector()]
 			for _, httpRule := range method.HttpRule {
 				if httpRule.HttpMethod != "OPTIONS" {
```

Comment:
NIT: replace with util.OPTIONS

Reply:
Will do in the next PR, then I won't be blocked.